airbyte-integrations/bases/base-normalization/normalization/transform_catalog/utils.py

```diff
@@ -16,76 +17,125 @@ def remove_jinja(command: str) -> str:
     return str(command).replace("{{ ", "").replace(" }}", "")
 
 
+# transform oneOf nested list ot plain types list
+def get_plain_list_from_one_of_array(one_of_definition) -> list:
+    one_of_property = one_of_definition[data_type.ONE_OF_VAR_NAME]
+    plain_datatypes_list = list()
+    for prop in one_of_property:
+        plain_datatypes_list.append(prop[data_type.REF_TYPE_VAR_NAME])
```

Comment:
@edgao In this case, it will take a String type by already existing logic. If the string is in the list, then other types would be ignored. As far as I see from Normalization logic -  it always takes a wider type for SQL column-> string